programs/sbf/Cargo.lock

```diff
@@ -7773,6 +7773,7 @@ dependencies = [
  "solana-feature-set",
  "solana-log-collector",
  "solana-program-runtime",
+ "solana-pubkey",
```

Comment:
nit: is change to `sbf` belong to this PR?

Reply:
yes, this came from running `scripts/cargo-for-all-lock-files.sh tree`. CI would fail without it